### PR TITLE
Add ability to run all paths in config easily

### PR DESCRIPTION
**Pull request by jackzampolin**

### CODE REVIEW

**Review by boojamya — APPROVED** [review]
This is awesome!

I think we should make a note that this is possible on the Readme.

Under Step 9, something like:
`If no path is specified, the relayer will attempt to relay on all configured paths`

File: cmd/start.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ func startCmd(a *appState) *cobra.Command {
 		Use:     "start path_name",
 		Aliases: []string{"st"},
 		Short:   "Start the listening relayer on a given path",
-		Args:    withUsage(cobra.MinimumNArgs(1)),
+		Args:    withUsage(cobra.MinimumNArgs(0)),
 		Example: strings.TrimSpace(fmt.Sprintf(`
```
Comment (mark-rushakoff) [inline cmd/start.go:42]: Examples could be updated here.

File: cmd/start.go [hunk 2]
```diff
@@ -47,16 +47,29 @@ $ %s start demo-path2 --max-tx-size 10`, appName, appName, appName)),
...
+					}
 
-				// collect unique chain IDs
-				chains[path.Src.ChainID] = nil
-				chains[path.Dst.ChainID] = nil
+					// collect unique chain IDs
+					chains[path.Src.ChainID] = nil
+					chains[path.Dst.ChainID] = nil
+				}
+			} else if len(args) == 0 {
```
Comment (mark-rushakoff) [inline cmd/start.go]: Just `else` is appropriate here.

File: cmd/start.go (hunk 1 above)
Comment (jackzampolin) [reply cmd/start.go:42]: Fixed examples!! Thank you @mark-rushakoff

**Pull request merged**